coverage.html

```diff
@@ -8140,6 +8140,31 @@
 // The maximum value is 1000, indicating a 1% fee. The minimum value is 0. https://xrpl.org/docs/references/protocol/transactions/types/ammcreate#ammcreate-fields
 const AMM_MAX_TRADING_FEE = 1000
 
+// Create a new Automated Market Maker (AMM) instance for trading a pair of assets (fungible tokens or XRP).
```

Comment:
I forgot to re-run `make coverage-unit` in my previous PR where I added some comments for AMMCreate and AMMDelete.